local_planner/src/nodes/planner_functions.cpp

```diff
@@ -399,19 +403,26 @@ void findFreeDirections(
 
           // Elevation index < 0
           if (i < 0 && j >= 0 && j < z_dim) {
-            a = -i;
-            b = z_dim - j - 1;
+            a = 0;
```

Comment:
This logic gets a bit complicated, hopefully when we take into account the widths we can make a simpler version, maybe by switching to cartesian coordinates and back.